core/src/elements/ons-back-button.js

```diff
@@ -74,15 +74,17 @@ class BackButtonElement extends BaseElement {
    *  [ja]バックボタンの見た目を指定します。[/ja]
    */
 
-  createdCallback() {
-    contentReady(this, () => {
-      if (!this.hasAttribute('_compiled')) {
-        this._compile();
-      }
+  constructor(self) {
+    self = super(self);
+
+    contentReady(self, () => {
+      self._compile();
```

Comment:
@frankdiox Yes, I don't think that is supported. We already extend `BaseElement` so I think we can add the ugly constructor there and then use `init` in the other components.